response_operations_ui/templates/respondent-search/search-respondents-results.html

```diff
@@ -0,0 +1,19 @@
+{% extends "layouts/base.html" %}
+{% block page_title %}Respondents | Survey Data Collection{% endblock %}
+
+{% block main %}
+<div role="main" id="main" class="page__main">
+    <div class="grid">
+        <div class="grid__col col-12@l">
+            {% include 'respondent-search/_inline-form.html' %}
+    
+            <h2 class="u-fs-l">{{ respondent_count }} respondents found.</h2>
+            <h3 class="u-mb-m">Displaying <strong>{{ first_index }} - {{ last_index }}</strong> of <strong>{{ respondent_count }}</strong> </h3>
+    
+            {% if respondents %}{% include 'respondent-search/_table.html' %}{% endif %}
+
+            {% if respondent_count > 25 %}{% include 'respondent-search/_pagination.html' %}{% endif %}
```

Comment:
probably shouldn't be hard-coded to `25`. This is a configurable parameter, isn't it?

Reply:
Good catch.